distribution/lib/Standard/Visualization/0.0.0-dev/src/Scatter_Plot.enso

```diff
@@ -100,19 +115,28 @@ type No_Fallback_Column
 ## PRIVATE
 
    Generates JSON that describes points data.
-Table.point_data : Vector
-Table.point_data self =
+Table.point_data : Vector -> Vector
+Table.point_data self all_fields =
     get_point_data field = field.lookup_in self . rename field.name . catch Any (_->Nothing)
     is_not_nothing x = case x of
         Nothing -> False
         _ -> True
-    columns = Point_Data.all_fields.map get_point_data . filter is_not_nothing
+    columns = all_fields.map get_point_data . filter is_not_nothing
     (0.up_to self.row_count).to_vector.map <| row_n->
         pairs = columns.map column->
             value = column.at row_n . catch_ Nothing
             [column.name, value]
         JS_Object.from_pairs pairs
 
+## PRIVATE
+
+   Generates JSON that describes points data.
+Table.multi_series_info : Number
+Table.multi_series_info self =
+    is_good c = c.is_numeric
+    columns = self.columns . filter is_good
+    columns.length
```

Comment:
I don't understand the name of this method.

It is supposed to be returning some "info" but that info is a number. Maybe `numeric_columns_count` would work better?

Also I'm not quite sure if the documentation comment on it is correct? It seems like copied from the method above.

Reply:
Yeah sorry I think I had initially named it thinking it would do something different and didn't go back to change it's name!